index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" /> 
+        <!-- Load icon library -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio page</title>
+</head>
+<body>
+    <section class="Portfolio">
+         <!-- The form -->
+    <form class="example" action="action_page.php">
+        <input type="text" placeholder="" name="search">
+        <button type="submit"><i class=""></i>Login</button>
+        <!-- About section -->
+        <textarea class="about" name="about" rows="3" cols="20">
+            My name is Fazila Abbasi.I am from Rahim Yar Khan.I am a Wordpress developer.I have completed my internship at CodeNinja.
+            I also have a 3 month experience of Data Annotation.I clear my Matric and FSC from Daanish School Rahim Yar Khan.
+            I clear my BSCS (2018-2022) from Govt Khawaja Fareed P/G college Rahim Yar Khan.
+            </textarea>
+        <!-- Experience section -->
+        <input class="experience" type="text" placeholder="Experience">
+        <!-- Footer section -->
+            <div class="footer">
+                <input type="text" placeholder="" name="search"> 
+                <a href="https://www.linkedin.com/in/fazila-abbasi-a07b18264/">  
+                    <button>Link to My LinkedIn Profile</button> 
+                  </a>
+            </div>
+        <!-- Sidebar section -->
+        
+        <div class="sidebar">
```

Comment:
![image](https://github.com/FazilaAbbasi/Fazila-HTML-CSS-Basics/assets/26036171/d496940d-2d6d-4a42-9b7f-d0eee8bfffc1)

try to center the avatar and your name horizontally in the box